src/helperFunctions/install.py

```diff
@@ -239,3 +240,7 @@ def load_main_config() -> configparser.ConfigParser:
         raise InstallationError('Could not load config at path {}'.format(config_path))
     config.read(str(config_path))
     return config
+
+
+def is_virtualenv() -> bool:
```

Comment:
Uargh. Short explanation? (StackOverflow post this was copied from suffices)

Reply:
https://stackoverflow.com/questions/1871549/determine-if-python-is-running-inside-virtualenv :sweat_smile: